pkg/nasm/config.h

```diff
@@ -160,8 +163,8 @@
 #define HAVE___BUILTIN_BSWAP16 1
 #define HAVE___BUILTIN_BSWAP32 1
 #define HAVE___BUILTIN_BSWAP64 1
-/* probe HAVE___BUILTIN_CHOOSE_EXPR */
-/* probe HAVE___BUILTIN_CLZ */
+#define HAVE___BUILTIN_CHOOSE_EXPR 1
+#define HAVE___BUILTIN_CLZ 1
```

Comment:
I think these should remain commented out. They are checked separately by probe: 
https://github.com/oasislinux/oasis/blob/master/probe/gen.lua#L13

Reply:
Thanks for catching this, @rnpnr!